### PR TITLE
Exposed Chat and Call latestErrors in Meeting Adapter

### PR DESCRIPTION
**Pull request by edwardlee-msft**
# What
Exposed Chat and Call latestErrors in Meeting Adapter

# Why
https://skype.visualstudio.com/SPOOL/_workitems/edit/2755199

# How Tested
Force triggering errors by throwing on events like Mute and seeing the errors in console log.

# Process & policy checklist
- [x] I have updated the project documentation to reflect my changes if necessary.
- [x] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

### CODE REVIEW

File: packages/react-composites/src/composites/MeetingComposite/adapter/AzureCommunicationMeetingAdapter.ts [hunk 1]
```diff
@@ -328,7 +329,10 @@ export class AzureCommunicationMeetingAdapter implements MeetingAdapter {
         this.chatAdapter.on('messageRead', listener);
         break;
       case 'error':
-        throw 'on(Error) not implemented yet.';
+        this.callAdapter.on('error', listener);
+        this.chatAdapter.on('error', listener);
+        // throw 'on(Error) not implemented yet.';
```
Comment (prprabhu-ms) [inline packages/react-composites/src/composites/MeetingComposite/adapter/AzureCommunicationMeetingAdapter.ts:334]: Remove commented line.

File: packages/react-composites/src/composites/MeetingComposite/adapter/AzureCommunicationMeetingAdapter.ts [hunk 2]
```diff
@@ -384,7 +388,10 @@ export class AzureCommunicationMeetingAdapter implements MeetingAdapter {
         this.chatAdapter.off('messageRead', listener);
         break;
       case 'error':
-        throw 'on(Error) not implemented yet.';
+        this.callAdapter.off('error', listener);
+        this.chatAdapter.off('error', listener);
+        // throw 'on(Error) not implemented yet.';
```
Comment (prprabhu-ms) [inline packages/react-composites/src/composites/MeetingComposite/adapter/AzureCommunicationMeetingAdapter.ts:393]: Remove commented line

**Review by prprabhu-ms — APPROVED** [review]

**Review by anjulgarg — APPROVED** [review]

File: packages/react-composites/src/composites/MeetingComposite/adapter/AzureCommunicationMeetingAdapter.ts (hunk 2 above)
Comment (edwardlee-msft) [reply packages/react-composites/src/composites/MeetingComposite/adapter/AzureCommunicationMeetingAdapter.ts:393]: Removed the comment

File: packages/react-composites/src/composites/MeetingComposite/adapter/AzureCommunicationMeetingAdapter.ts (hunk 1 above)
Comment (edwardlee-msft) [reply packages/react-composites/src/composites/MeetingComposite/adapter/AzureCommunicationMeetingAdapter.ts:334]: Removed commented line

**Pull request merged**